docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+  frozen_flame:
+    container_name: frozenFlame
+    build: .
+    image: ragingdave/FrozenFlameServer:latest
+    network_mode: "host"
+    volumes:
+      - "/path/to/config:/config"
+    #ports:
+    #    - 7779:7779
+    #    - 7780:7780/udp
+    #    - 27017:27017/udp
+    expose:
```

Comment:
I don't think from a docker perspective, we really need to deviate from the default ports as you can map them however you wish. With that I think we can remove the SERVER_PORT/SERVER_QUERY_PORT/RCON_PORT as configurable because of the nature of docker/port handling.

Reply:
It depends of how do you use docker. I don't know if my method is the best, but my docker container share the host ports.
I know some people prefers use port transfer.
But I prefers for some services use directly the hosts ports, I gain speed of container start/stop, latency to the service and processor usage.